spanner/opencensus/pom.xml

```diff
@@ -59,6 +59,10 @@
       <artifactId>opencensus-contrib-grpc-metrics</artifactId>
       <version>${opencensus.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-census</artifactId>
```

Comment:
Would you add source code comment when a user wants this?

Reply:
Added.